tests/testthat/test-Data-methods.R

```diff
@@ -0,0 +1,9 @@
+# as.list ----
+
+test_that("Coercion creates the expected `List` object", {
```

Comment:
```suggestion
test_that("Coercion creates the expected list", {
```
(we typically don't use backticks in the test labels)